### PR TITLE
added html page Added Helloworld message

### PR DESCRIPTION
**Pull request by Himabindujunnuthula**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<html lang="en">
+<head>
+    
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Helloworld</h1>
```
Comment (Bashaboina) [inline index.html]: Hello world message should contain space.

**Review by Bashaboina — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Bashaboina) [reply index.html]: Ok I will work on that.

Comment (Himabindujunnuthula) [reply index.html]: I will take care of this.

**Review by Bashaboina — APPROVED** [review]
All good.

**Pull request merged**